crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -255,7 +266,7 @@ impl Entities {
             // Allocate from the freelist.
             let id = self.pending[(n - 1) as usize];
             Entity {
-                generation: self.meta[id as usize].generation,
+                generation: self.meta[id as usize].generation.unwrap(), // Safe, meta from pending list, so generation is valid
```

Comment:
The use of the term safe here is peculiar - an unwrap failing/triggering is never UB.

I'd probably change this to use `expect`